extension/content.js

```diff
@@ -7,6 +7,11 @@ const getUsername = () => $('meta[name="user-login"]').attr('content');
 
 function linkifyBranchRefs() {
 	$('.commit-ref').each((i, el) => {
+		if ($(el).children().eq(0).text() === 'unknown repository') {
```

Comment:
Let's cache the value of `$(el)` so we don't have to hit the DOM twice for the same element.

`const $el = $(el);`